### PR TITLE
MINIFICPP-1321 Add documentation to MergeContent's attribute strategy

### PR DESCRIPTION
**Pull request by lordgamez**
Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [ ] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Review by szaszm — APPROVED** [review]

File: PROCESSORS.md [hunk 1]
```diff
@@ -757,7 +757,8 @@ Merges a Group of FlowFiles together based on a user-defined strategy and packag
 In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
 
 | Name | Default Value | Allowable Values | Description | 
-| - | - | - | - | 
+| - | - | - | - |
+|Attribute Strategy|Keep Only Common Attributes|Keep Only Common Attributes<Br>Keep All Unique Attributes<Br>|Determines which FlowFile attributes should be added to the bundle. If 'Keep All Unique Attributes' is selected, any attribute on any FlowFile that gets bundled will be kept unless its value conflicts with the value from another FlowFile. If 'Keep Only Common Attributes' is selected, only the attributes that exist on all FlowFiles in the bundle, with the same value, will be preserved.|
```
Comment (fgerlits) [inline PROCESSORS.md:761]: What happens to an attribute which exists in two flow files with different values if 'Keep All Unique Attributes' is selected?  Will both be discarded, or will one of them be kept?

Comment (lordgamez) [reply PROCESSORS.md:761]: Both will be discarded

Comment (fgerlits) [reply PROCESSORS.md:761]: I would add that clarification to the description: "... unless its value conflicts with the value from another FlowFile (in which case neither, or none, of the conflicting attributes will be kept)."

Comment (lordgamez) [reply PROCESSORS.md:761]: Shall I also update the property in MergeContent.cpp as currently it has the same description?

Comment (fgerlits) [reply PROCESSORS.md:761]: I would, unless there is a length limit in the C2 GUI.

**Review by fgerlits — APPROVED** [review]

**Pull request closed without merging**